kafka_utils/kafka_consumer_manager/commands/list_groups.py

```diff
@@ -53,7 +54,9 @@ def print_groups(cls, groups, cluster_config):
     @classmethod
     def run(cls, args, cluster_config):
         groups = set()
-        kafka_groups = cls.get_kafka_groups(cluster_config)
+        kafka_groups = cls.get_kafka_groups(
+            cluster_config, args.use_admin_client,
```

Comment:
Just a note: this doesn't match PEP8 (one arg per line in multi-line functions) but that's fine, since we don't have it as a pre-commit hook...

Maybe some day I'll add it to our pre-commit hooks and get constant style enforcement.

Reply:
I was following black's guideline, but we've been using PEP8 in this codebase. Good point :)